sdk/docs/core/CHANGELOG.md

```diff
@@ -2,10 +2,20 @@
 
 
 ## 1.0.0-preview.4 (Unreleased)
-
+- `az_context`:
+  - `key` and `value` are `const`.
+  - `az_context_with_expiration()` is renamed to `az_context_create_with_expiration()`.
+  - `az_context_with_value()` is renamed to `az_context_create_with_value()`.
+- `az_credential_client_secret_init()` now takes fourth parameter, `authority`.
+- `az_http_policy_retry_options`:
+  - `status_codes` now should be terminated by `AZ_HTTP_STATUS_CODE_END_OF_LIST`.
```

Comment:
I don't think this is a breaking change to the end user, we are just adding another value.

If we were removing `none` I would have mentioned it, but I think we can remove this as it is minor implementation detail.

Reply:
I think it is - if you used to have a list terminated by NONE, everything worked correctly, now you HAVE to replace NONE with END_OF_LIST, because otherwise it is going to read the entire memory until it manages to read -1.

Reply:
Oh right, this list of status codes is user provided. Gotcha.